README.md

```diff
@@ -93,9 +93,9 @@ In **PowerShell** or in a **Command Prompt** window:
         sqlite-devel.x86_64 readline-devel.x86_64 bzip2-devel.x86_64
     ```
 
-- **OS X**
+- **macOS**
 
-  Most problems on OS X have been with loading and linking OpenSSL and zlib. The following commands install necessary packages and tell the Python installer where to find them.
+  Most installation problems on macOS have involved loading and linking `OpenSSL` and `zlib`. The following command installs the necessary packages and tell the Python installer where to find them.
```

Comment:
`tell` should be `tells`, although `lets` is probably better.

Reply:
Good catch.

Reply:
Got it. Updated.